common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -113,105 +123,32 @@ case class LogMarkerToken(component: String, action: String, state: String) {
-    val INVOKER_FETCH_ACTION_DONE = LogMarkerToken(invoker, fetchAction, finish)
-    val INVOKER_FETCH_ACTION_FAILED = LogMarkerToken(invoker, fetchAction, error)
-
-    private val fetchAuth = "fetchAuth"
-    val INVOKER_FETCH_AUTH_START = LogMarkerToken(invoker, fetchAuth, start)
-    val INVOKER_FETCH_AUTH_DONE = LogMarkerToken(invoker, fetchAuth, finish)
-    val INVOKER_FETCH_AUTH_FAILED = LogMarkerToken(invoker, fetchAuth, error)
-
-    private val getContainer = "getContainer"
-    val INVOKER_GET_CONTAINER_START = LogMarkerToken(invoker, getContainer, start)
-    val INVOKER_GET_CONTAINER_DONE = LogMarkerToken(invoker, getContainer, finish)
-
-    val INVOKER_CONTAINER_INIT = LogMarkerToken(invoker, "initContainer", start)
-
-    private val activationRun = "activationRun"
-    val INVOKER_ACTIVATION_RUN_START = LogMarkerToken(invoker, activationRun, start)
-    val INVOKER_ACTIVATION_RUN_DONE = LogMarkerToken(invoker, activationRun, finish)
-
-    val INVOKER_ACTIVATION_START = LogMarkerToken(invoker, activation, start)
-    val INVOKER_ACTIVATION_DONE = LogMarkerToken(invoker, activation, finish)
-    val INVOKER_ACTIVATION_ERROR = LogMarkerToken(invoker, activation, error)
-
-    private val recordActivation = "recordActivation"
-    val INVOKER_RECORD_ACTIVATION_START = LogMarkerToken(invoker, recordActivation, start)
-    val INVOKER_RECORD_ACTIVATION_DONE = LogMarkerToken(invoker, recordActivation, finish)
-
-    val DATABASE_CACHE_HIT = LogMarkerToken(database, "cacheHit", start)
-    val DATABASE_CACHE_MISS = LogMarkerToken(database, "cacheMiss", start)
-
-    private val saveDocument = "saveDocument"
-    val DATABASE_SAVE_START = LogMarkerToken(database, saveDocument, start)
-    val DATABASE_SAVE_DONE = LogMarkerToken(database, saveDocument, finish)
-    val DATABASE_SAVE_ERROR = LogMarkerToken(database, saveDocument, error)
-
-    private val deleteDocument = "deleteDocument"
-    val DATABASE_DELETE_START = LogMarkerToken(database, deleteDocument, start)
-    val DATABASE_DELETE_DONE = LogMarkerToken(database, deleteDocument, finish)
-    val DATABASE_DELETE_ERROR = LogMarkerToken(database, deleteDocument, error)
-
-    private val getDocument = "getDocument"
-    val DATABASE_GET_START = LogMarkerToken(database, getDocument, start)
-    val DATABASE_GET_DONE = LogMarkerToken(database, getDocument, finish)
-    val DATABASE_GET_ERROR = LogMarkerToken(database, getDocument, error)
-
-    private val queryView = "queryView"
-    val DATABASE_QUERY_START = LogMarkerToken(database, queryView, start)
-    val DATABASE_QUERY_DONE = LogMarkerToken(database, queryView, finish)
-    val DATABASE_QUERY_ERROR = LogMarkerToken(database, queryView, error)
+
+    // Time of the activation in controller until it is delivered to Kafka
+    val CONTROLLER_ACTIVATION = LogMarkerToken(controller, activation, start)
+    val CONTROLLER_BLOCKING_ACTIVATION = LogMarkerToken(controller, "blockingActivation", start)
```

Comment:
`CONTROLLER_BLOCKING_ACTIVATION -> CONTROLLER_ACTIVATION_BLOCKING`?
Also I suggest shorter marker names - if you can make them all fixed size, then you can ignore my request to move the marker to the end of the log message above.